src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -397,15 +399,6 @@ private void hutDecClicked(@NotNull final Button button)
      */
     private List<String> getStyles()
     {
-        try
```

Comment:
Whatever method I will implement to add new schematic on the fly will trigger a refresh of this list.